<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MalzemeKalemiDialog</class>
 <widget class="QDialog" name="MalzemeKalemiDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Kategoriler</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListView" name="listView_kategori"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_deleteKategori">
            <property name="text">
             <string>Sil</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_refreshKategori">
            <property name="text">
             <string>Yenile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_NewKategori">
            <property name="text">
             <string>Yeni Kategori</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Malzemeler</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListView" name="listView_malzeme"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_deleteMalzeme">
            <property name="text">
             <string>Sil</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_refreshMalzeme">
            <property name="text">
             <string>Yenile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_NewMalzeme">
            <property name="text">
             <string>Yeni Malzeme</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Malzeme Detayları</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Kategori</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit_MalzemeOid">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Birimi</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Adı</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Kodu</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>KDV</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit_MalzemeName"/>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit_MalzemeBirimi"/>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_MalzemeComboBox"/>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="comboBox_MalzemeKategori"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_SaveMalzeme">
          <property name="text">
           <string>Kaydet</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string>status</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
